### PR TITLE
OPML export: Link to documentation

### PR DESCRIPTION
**Pull request by math-GH**
Ref. https://github.com/FreshRSS/FreshRSS/issues/6433#issuecomment-2099308728

Changes proposed in this pull request:

add link to documentation
grafik

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/en/sub.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
 		'export-as-opml' => array(
 			'download' => 'Download',
-			'help' => 'XML file',
+			'help' => 'XML file (restricted data. <a href="https://freshrss.github.io/FreshRSS/en/developers/OPML.html" target="_blank">See documentation</a>)',
```
Comment (math-GH) [inline app/i18n/en/sub.php]: Is it a good wording?
grafik

Comment (Alkarex) [reply app/i18n/en/sub.php]: I am not sure to understand what you mean by *restricted data*

Comment (math-GH) [reply app/i18n/en/sub.php]: I want to highlight that not all data from above will be exported

Comment (Alkarex) [reply app/i18n/en/sub.php]: Ah, then maybe *subset* or *partial*  or *limited* or something like that

File: app/i18n/fr/sub.php [hunk 2]
```diff
@@ -68,7 +68,7 @@
 		'error' => 'Ce flux a rencontré un problème. Veuillez vérifier qu’il est toujours accessible puis actualisez-le.',
 		'export-as-opml' => array(
 			'download' => 'Télécharger',
-			'help' => 'Fichier XML',
+			'help' => 'Fichier XML (data subset. <a href="https://freshrss.github.io/FreshRSS/en/developers/OPML.html" target="_blank">See documentation</a>)',	// DIRTY
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
			'help' => 'Fichier XML (données partielles. <a href="https://freshrss.github.io/FreshRSS/en/developers/OPML.html" target="_blank">Voir documentation</a>)',
```

**Pull request merged**